### DISCUSSION TITLE
Open a warning dialog before opening x-number of folders when clicking 'show files'

### DISCUSSION THREAD

**Discussion by BiggRanger** [TumblThreeApp/TumblThree#275] [Ideas]
If a user has 30+ blogs, and accidentally clicks 'show files' while they are all highlighted (instead of clicking 'add to queue'), they will end up with 30+ explorer windows popping up.

A simple warning, "Are you sure you want to open 55 blog folders?", if more than 5 blogs is selected when clicking 'show files' would be nice, then the user could cancel the operation.

**Comment by thomas694** [comment]
Thanks for your suggestion. That sounds like a good idea.

To be honest, I had it myself before, but as my taskbar settings are set to "group taskbar buttons" I only had to right-click and close all at once.
